joomla/joomla-cms
# Revert #19977: Plugins field behavior

Pull Request for Issue # .

### Summary of Changes

PR #19977 wasn't tested properly. As a result, it introduced unwanted behavior when saving form. The change prevents default value from being saved when it is set as a custom option in XML form.

Additionally, it adds check for a hardcoded language constant. This is not correct because the constant is added through custom option in XML form and could be anything.

### Testing Instructions

Enable a Captcha plugin (e.g. reCaptcha).
Select the plugin as default Captcha plugin in Global Configuration and save.
Disable the plugin.
Go to Global Configuration and save again.
a) Open username reminder or password reset page
b) Check Captcha setting value (either in configuration.php or through JFactory::getApplication()->get('captcha')).

### Expected result

a) No traces of captcha.
b) `0`

### Actual result

a) Captcha label shown in form, indicating that Captcha plugin is set in configuration.
b) `recaptcha`

### Documentation Changes Required
No.
